Description: function AC_CHECK_FILE of autoconf is not support for crossbuild.
	This patch replace AC_CHECK_FILE by AS_IF function.
	AS_IF feature is just similar to the AC_CHECK_FILE, AS_IF check the first
	param exits and return a zero status then run shell code in second param.

diff --git a/pixz-1.0.6/debian/patches/0002-configure-ac-crossbuild.patch b/pixz-1.0.6/debian/patches/0002-configure-ac-crossbuild.patch
new file mode 100644
index 0000000..3492175
--- /dev/null
+++ b/pixz-1.0.6/debian/patches/0002-configure-ac-crossbuild.patch
@@ -0,0 +1,15 @@
+diff --git a/pixz-1.0.6/configure.ac b/pixz-1.0.6/configure.ac
+index 4cb56bc..4ec96f5 100644
+--- a/pixz-1.0.6/configure.ac
++++ b/pixz-1.0.6/configure.ac
+@@ -16,9 +16,7 @@ AC_PROG_CC_STDC
+ # Check for a2x only if the man page is missing, i.e. we are building from git. The release tarballs
+ # are set up to include the man pages. This way, only people creating tarballs via `make dist` and
+ # people building from git need a2x as a dependency.
+-AC_CHECK_FILE(
+-  [src/pixz.1],
+-  [],
++AS_IF([test ! -r "src/pixz.1"],
+   [
+     AC_ARG_WITH(
+       [manpage],
diff --git a/pixz-1.0.6/debian/patches/series b/pixz-1.0.6/debian/patches/series
index 763aa33..d3d9036 100644
--- a/pixz-1.0.6/debian/patches/series
+++ b/pixz-1.0.6/debian/patches/series
@@ -1,3 +1,3 @@
 # exported from git by git-debcherry
 0001-option-to-not-build-man-page.patch
+0002-configure-ac-crossbuild.patch
-- 
2.15.1
